superdesk/data_updates/00007_20180321-092824_archive.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Author  : tomas
+# Creation: 2018-03-21 09:28
+
+import os
+import subprocess
+import json
+from superdesk.commands.data_updates import DataUpdate
+
+def get_updated_editor_state(editor_state):
+    try:
+        p1 = subprocess.Popen(["echo", json.dumps(editor_state)], stdout=subprocess.PIPE)
+        p2 = subprocess.Popen(["node", os.path.abspath('../env/src/superdesk-core/superdesk/data_updates/00007_20180321-092824_archive.dist.js')], stdin=p1.stdout, stdout=subprocess.PIPE)
+        p1.stdout.close()  # Allow p1 to receive a SIGPIPE if p2 exits.
+        output,err = p2.communicate()
+
+        return json.loads(output.decode("utf-8"))
+
+    except Exception as exception:
```

Comment:
you don't need to put `as exception` if you don't use it (in fact, flake8 tests wont like it).
Also it's good practice to put explicitly the Exceptions you are expecting, but as we are on a one shot upgrade script, it's probably not a big deal here.

Reply:
I wasn't sure if it's required, removed them.